cmd/main.go

```diff
@@ -139,7 +140,11 @@ func main() {
 		log.Infof(nil, "Service Revision %s built on %s", configuration.Commit, configuration.BuildTime)
 		log.Infof(nil, "Listening on %q...", configuration.HTTPAddress)
 		if err := srv.HTTPServer().ListenAndServe(); err != nil {
-			log.Error(nil, err, err.Error())
+			if errors.Is(err, http.ErrServerClosed) {
+				log.Info(nil, err.Error())
```

Comment:
Should we add some additional context to this log? eg. `The server has closed, this is expected when server shutdown has been initiated.`

Reply:
+1 @rajivnathan

Reply:
makes sense, see [6a877c4](https://github.com/codeready-toolchain/registration-service/pull/401/commits/6a877c4219212c13f1876e8b6d458b4380169797)

thanks for your suggestion!